docs/src/snap/howto/refresh-certs.md

```diff
@@ -0,0 +1,103 @@
+# Refreshing Kubernetes Certificates
+
+Keeping your {{product}} cluster secure and functional requires
+regularly refreshing its certificates. Certificates in Kubernetes ensure
+secure communication between the various components of the cluster. If
+these certificates expire, it can lead to communication failures, disrupted
+services, and potential security risks. This how-to will walk you through
+the steps to refresh the certificates for both control plane and worker
+nodes in your {{product}} cluster.
+
+## What you will need
+
+- A running {{product}} cluster
+
+```{note} To refresh the certificates in your cluster, make sure it was
+initially set up with self-signed certificates during the bootstrap process.
```

Comment:
I think this will only add complexity to the guide. Without using `openssl` tricks, it would be difficult to determine if the certificates are self signed or not. The administrator should already know if the cluster was setup with custom CA, as they should have provided custom certs during the bootstrap and join phases.

Reply:
That makes sense, thanks Mateo!